### ISSUE TITLE
LXD init breaks the mist.io shell

### ISSUE THREAD

**Issue by mmmccain** [mistio/mist-ce#1019]
I stood up Ubuntu 20.04 from mist, opened the shell and executed lxd init. About half way through the script it asks for an IP address to be entered. The mist shell does not display this correctly and it will not accept even the default (CR). It just breaks the script. 

This is very easy to reproduce and it is consistent. 

Work around is to ssh to install lxd. This is the only thing I have seen the mist shell have a problem with so far.

**Comment by Eis-D-Z** [comment]
I will kindly ask for some clarifications. 
The Ubuntu machine was hosted on Linode?
By not being the IP displayed correctly, it means that you typed eg. 192.149.233.22 and something else showed on screen?
I am understanding that because the IP that got typed did not go through properly, the script also crashed because it expected to receive an IP but it got something else, correct?

**Comment by mmmccain** [comment]
I do wish I had grabbed a screen shot. 

This is an STTY issue in SSH. It is receiving a set of characters from the from the shell script "lxd init" that it can not interrupt correctly.  Once the shell receives these characters, which makes the line no longer fully readable, it will not receive any text input of any kind, not even a carriage return. ...and yes, it will not let you type an IP address. Nothing is shown on the screen.

**Comment by Eis-D-Z** [comment]
Indeed, it is an issue with encoding the ".  Until this is resolved you can use this opportunity to check the  SSH from our [CLI].(https://github.com/mistio/mist-cli).

**Comment by Eis-D-Z** [comment]
We found the issue and we are fixing it!
